src/averell/utils.py

```diff
@@ -288,9 +283,9 @@ def read_features(corpus_folder):
 
 def pretty_string(text, n_words):
     """Add a line break every number of words into a text to create multiline
-    cells to use in `tabulate_corpora`
+    cells to use in :py:func:`~averell.utils.get_main_corpora_info`
 
-    :param text: String to be converted
+    :param text: String to be divided
     :param n_words: Number of words to add a line break after
```

Comment:
`n_word` is a term to refer to the word "nigger", not sure if this is appropiate :), maybe change it to something else?

Reply:
Sorry, changed to num_words